### PR TITLE
Update update-images.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Add proper syntax to answers in the RQ

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/web/html/core-html-elements/update-images.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/web/html/core-html-elements/update-images.md?ref=92dafa6d11648ff92776a8a1a02128888d4fea28)

File: web/html/core-html-elements/update-images.md [hunk 1]
```diff
@@ -90,12 +90,12 @@ If an image does not have an assigned height and width using HTML or CSS, how wi
 
 ???
 
-* `img src="image.jpg" alt="Image Description"`
+* `<img src="image.jpg" alt="Image Description">`
```
Comment (kapnobatai137) [inline web/html/core-html-elements/update-images.md]:
```suggestion
* <img src="image.jpg" alt="Image Description">
```

File: web/html/core-html-elements/update-images.md [hunk 2]
```diff
@@ -90,12 +90,12 @@ If an image does not have an assigned height and width using HTML or CSS, how wi
...
 * `alt`
 * `.img`
```
Comment (kapnobatai137) [inline web/html/core-html-elements/update-images.md]:
```suggestion
* alt
* .img
```

File: web/html/core-html-elements/update-images.md [hunk 3]
```diff
@@ -90,12 +90,12 @@ If an image does not have an assigned height and width using HTML or CSS, how wi
...
 * At its original image size.
-* `image src="image.jpg" alt="Image Description"`
-* `img alt="image.jpg" src="Image Description"`
+* `<image src="image.jpg" alt="Image Description">`
+* `<img alt="image.jpg" src="Image Description">`
```
Comment (kapnobatai137) [inline web/html/core-html-elements/update-images.md]: no need for backticks as the answers already display as if enclosed by backticks
image

```suggestion
* <image src="image.jpg" alt="Image Description">
* <img alt="image.jpg" src="Image Description">
```

File: web/html/core-html-elements/update-images.md [hunk 4]
```diff
@@ -90,12 +90,12 @@ If an image does not have an assigned height and width using HTML or CSS, how wi
...
 * `desc`
 * `.png`
```
Comment (kapnobatai137) [inline web/html/core-html-elements/update-images.md]:
```suggestion
* desc
* .png
```

**Pull request merged**